packages/@aws-cdk/cloudformation-include/README.md

```diff
@@ -19,6 +19,8 @@ It can be thought of as an extension of the capabilities of the
 
 Assume we have a file `my-template.json`, that contains the following CloudFormation template:
 
+We can also use YAML templates, instead of JSON.
```

Comment:
This should be below the JSON example , otherwise it reads really weird :).

Maybe something like:

```
Assume we have a file with an existing template. It could be in JSON format, in a file `my-template.json`:

JSON example file here...

Or it can be in YAML format, in a file `my-template.yaml`:

YAML example file here....
```

It can be included in a CDK application with the following code:

```ts
new cfn_inc.CfnInclude(this, 'Template', {
  templateFile: 'my-template.json', // or 'my-template.yaml' if your file is in that format
});
```